modules/audio_device/audio_device_impl.cc

```diff
@@ -664,7 +664,6 @@ int32_t AudioDeviceModuleImpl::MinMicrophoneVolume(uint32_t* minVolume) const {
   return 0;
 }
 
-// RingRTC changes to add some useful logging
```

Comment:
So the logging is now upstream as well? Or maybe we had bumped this up to LS_WARN, but no longer?

Reply:
Yeah, in this file, below this comment, there were no changes, no logging changes, no changes of any kind actually from us.